blant.c

```diff
@@ -653,28 +653,16 @@ static SET *SampleGraphletMCMC(SET *V, int *Varray, GRAPH *G, int k) {
 	//Our queue now contains k distinct nodes. Fill the set V and array Varray with them
 	int num, numNodes = 0, i;
 	SetEmpty(V);
-	for (i = 0; i < XLQ->length; i++) {
+	for (i = XLQ->length-1; i >= 0; i++) {
```

Comment:
Are you absolutely sure that i should be incremented in this loop?  Looks like it should be a decrement since the loop condition is i >= 0.

Reply:
my mistake